### PR TITLE
radicle-node: 1.0.0-rc.11 -> 1.0.0-rc.12

### PR DESCRIPTION
**Pull request by gador**
## Description of changes

https://radicle.zulipchat.com/#narrow/stream/409174-announcements/topic/radicle.201.2E0.2E0-rc.2E12

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by lorenzleutgeb — APPROVED** [review]

**Comment by wegank** [comment]
Looks like there are some extra tests to skip?

**Comment by gador** [comment]
> Looks like there are some extra tests to skip?

Why? And which?

Package builds on my end as it is

**Comment by amesgen** [comment]
@gador See ofborg: https://logs.ofborg.org/?key=nixos/nixpkgs.323862&attempt_id=b979083d-da56-42d1-aea8-2c22a9453a4a

I can also reproduce this locally (x86_64-linux), maybe you are on a darwin machine? (ofborg builds for darwin succeed just fine)

**Comment by gador** [comment]
Ah I see. No this seems to be a flaky test which sometimes succeeds.

I'll amend the pr to exclude it

**Review by amesgen — APPROVED** [review]

**Comment by wegank** [comment]
Still failing on aarch64-linux: https://logs.ofborg.org/?key=nixos/nixpkgs.323862&attempt_id=25c716cb-2ac5-4b2a-a228-9c99d402a2b6

**Comment by gador** [comment]
@wegank That's really interesting.  I don't know why, but tests fail for me regularly when trying to compile to `aarch64-linux`, but every time its a different test that fails. Also, it is one and only one test which times out and all other pass.  
I decided to just disable testing on `aarch64` .
@lorenzleutgeb what do you think?

File: pkgs/by-name/ra/radicle-node/package.nix [hunk 1]
```diff
@@ -16,23 +16,24 @@
...
 
   nativeBuildInputs = [ asciidoctor installShellFiles makeWrapper ];
   nativeCheckInputs = [ git ];
   buildInputs = lib.optionals stdenv.buildPlatform.isDarwin [
     darwin.apple_sdk.frameworks.Security
   ];
 
-  doCheck = stdenv.hostPlatform.isLinux;
+  # tests regularly time out on aarch64
+  doCheck = stdenv.buildPlatform.isx86;
```
[the remark this answers is not in the document]
Comment (gador) [reply pkgs/by-name/ra/radicle-node/package.nix:36]: oh boy.. Thanks for the catch

**Review by lorenzleutgeb — APPROVED** [review]

**Comment by lorenzleutgeb** [comment]
I don't like backpedaling on the tests. Hopefully I can contribute some fixes upstream soon.

**Pull request merged**